Configuration/Geometry/python/dict2023Geometry.py

```diff
@@ -853,6 +853,10 @@
             'from Geometry.MTDNumberingBuilder.mtdTopology_cfi import *',
             'from Geometry.MTDGeometryBuilder.mtdGeometry_cfi import *',
             'from Geometry.MTDGeometryBuilder.idealForDigiMTDGeometry_cff import *',
+            'mtdParameters.vitems[0].subdetPars[8] = 1 #bar geometry',

```

Comment:
will these settings (or other settings) be needed for the new geometry variations proposed in #24854?

Reply:
Yes, but be aware this is a kludge to get things more or less running.
I have a plan to transition the topology classes for BTL to use the sim geometry layout directly, but it'll come later and is not absolutely necessary for tracking to work.

Reply:
@lgray again this seems not needed any more